msgraph-developer-proxy/ProxyHost.cs

```diff
@@ -9,24 +9,29 @@ namespace Microsoft.Graph.DeveloperProxy;
 
 internal class ProxyHost {
     private Option<int> _portOption;
+    private Option<LogLevel> _logLevelOption;
 
     public ProxyHost() {
         _portOption = new Option<int>("--port", "The port for the proxy server to listen on");
         _portOption.AddAlias("-p");
         _portOption.ArgumentHelpName = "port";
         _portOption.SetDefaultValue(8000);
+
+        _logLevelOption = new Option<LogLevel>("--logLevel", "Level of messages to log");
```

Comment:
This should be nullable and not provide a default, otherwise with the current implementation we'll get `LogLevel.Info` if the config is for `warn` and the user doesn't pass a command line option